gems/aws-sdk-core/lib/aws-sdk-core/rest/request/querystring_builder.rb

```diff
@@ -49,8 +49,9 @@ def build_part(shape_ref, param_value)
               raise NotImplementedError, msg
             end
           when ListShape
-            if StringShape === shape_ref.shape.member.shape
-              list_of_strings(shape_ref.location_name, param_value)
+            if StringShape === shape_ref.shape.member.shape ||
+               IntegerShape === shape_ref.shape.member.shape
```

Comment:
Should we support all of the primitive shapes here as Smithy does (eg, boolean, float, ect).

Reply:
Probably, depends on what the ask is (awaiting more info)